src/Api/halls/api.js

```diff
@@ -0,0 +1,59 @@
+import axios from '../../config/axios';
+
+export const fetchHalls = async () => {
+  let halls = [];
+  await axios
+    .get('/halls')
+    .then((response) => {
+      if (response.status === 200) {
+        halls = response.data;
+      }
+    })
+    .catch((e) => console.log(e.response.data));
+  return halls;
+};
+
+// eslint-disable-next-line no-return-await
+export const addHall = async (data, setClose) => {
+  await axios
+    .post('/admin/halls', data)
+    .then((response) => {
+      if (response.status === 200) {
+        setClose(true);
+      }
+    })
+    .catch((e) => console.log(e.response.data));
+};
+
+// eslint-disable-next-line no-return-await
+export const editHall = async (id) => await axios
+  .get(`/admin/halls/${id}`)
+  .then((response) => {
+    if (response.status === 200) {
+      return response.data;
+    }
+    return null;
+  })
+  .catch((e) => console.log(e.response.data));
+
+// eslint-disable-next-line no-return-await
+export const updateHall = async (data) => await axios
+  .patch('/admin/halls', data)
+  .then((response) => {
+    if (response.status === 200) {
+      return response;
+    }
+    return false;
+  })
+  .catch((e) => console.log(e.response.data));
+
+// eslint-disable-next-line no-return-await
+export const removeHall = async (id) => await axios
+  .delete(`/admin/halls/${id}`)
+  .then((response) => {
+    if (response.status === 200) {
+      return response.data;
+    }
+    return null;
+  })
+  .catch((e) => console.log(e.response.data));
```

Comment:
- [ ] i think this file should be place in` Api/admins/halls/api.js ` instead of [Api/halls/api.js](https://github.com/ichala/short-let-frontend/pull/21/files#diff-590064c3ced00566e0cdab20820c6d57e8b12a14f945bdf24960db73e7789dd8) to make the files more organized and clean